src/confd/src/ietf-interfaces.c

```diff
@@ -1889,6 +1889,10 @@ static sr_error_t netdag_gen_iface(sr_session_ctx_t *session, struct dagger *net
 	if (err)
 		goto err_close_ip;
 
+	/* ifAlias, should skip for container-network types */
+	attr = lydx_get_cattr(cif, "description");
+	fprintf(ip, "link set alias \"%s\" dev %s\n", attr ?: "", ifname);
```

Comment:
No length check, resulting in confd crash:
admin@ix-00-00-00:/config/> 
admin@ix-00-00-00:/config/> set interface e1 description "hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej he
j hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej he
j hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej he
j hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej he
j hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej hej he
j hej hej hej hej hej hej"
admin@ix-00-00-00:/config/> leave 
Error: EV ORIGIN: SHM event "done" ID 3 processing timed out.
Error: Can't commit to running-config
admin@ix-00-00-00:/config/> 
admin@ix-00-00-00:/> 
this should be added to the YANG model.

The SNMP MIB has limitations OctetString (0...64)
Linux supports more, but if it is intended to use in IF-MIB 64 is a sane max value.

Reply:
Fixed, deviation for string length added to infix-interfaces.yang.